include/powsybl/stdcxx/time.hpp

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_STDCXX_TIME_HPP
+#define POWSYBL_STDCXX_TIME_HPP
+
+#include <ctime>
+
+namespace stdcxx {
+
+std::tm tm();
```

Comment:
you should rename this function to `make_tm` or something like this to avoid confusion between the function's name and the type's name. What do you think?